lambdas/tdr-draft-metadata-checks/src/test/scala/uk/gov/nationalarchives/tdr/draftmetadatachecks/LambdaSpec.scala

```diff
@@ -207,75 +231,46 @@ class LambdaSpec extends ExternalServicesSpec {
     checkFileError("json/error-file-invalid-description-with-alternate-description.json")
   }
 
-  private def checkFileError(errorFile: String) = {
+  "handleRequest" should "return correct response for a successful validation" in {
+    authOkJson()
+    graphqlOkJson(filesWithUniquesAssetIdKeyResponse = filesWithUniquesAssetIdKeyResponse(fileTestData))
```

Comment:
Can you remove some API stubs from `graphqlOkJson` method, which are not being used?